server/prisma/schema.prisma

```diff
@@ -0,0 +1,18 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "sqlite"
+  url      = env("DATABASE_URL")
+}
+
+model User {
+  id       Int      @id @default(uuid())
+  email    String   @unique
+  username String?
+  name     String
```

Comment:
I'm guessing this represent full name so to be more descriptive change the field name to fullName.